### PR TITLE
ElectronicArts.EAapp version 12.218.0.5475

### PR DESCRIPTION
**Pull request by Tiompkin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110615&drop=dogfoodAlpha

### CODE REVIEW

**Comment by ItzLevvie** [comment]
[Policy] Needs-CLA

**Comment by ItzLevvie** [comment]
[Policy] Needs-Author-Feedback

File: manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.locale.it-IT.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElectronicArts.EAapp
+PackageVersion: 12.218.0.5475
+PackageLocale: it-IT
+Publisher: Electronic Arts
+PackageName: EA app
```
Comment (stephengillie) [inline manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.locale.it-IT.yaml:8]:
```suggestion
PrivacyUrl:   http://www.ea.com/legal
PackageName: EA app
PackageUrl: https://www.ea.com/ea-app
```

File: manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ElectronicArts.EAapp
```
Comment (Trenly) [inline manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.installer.yaml:4]:
```suggestion
PackageIdentifier: ElectronicArts.EADesktop
```
This package already exists under the EADesktop name, this new version should be added there

File: manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.locale.it-IT.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElectronicArts.EAapp
```
Comment (Trenly) [inline manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.locale.it-IT.yaml:4]:
```suggestion
PackageIdentifier: ElectronicArts.EADesktop
```

File: manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: ElectronicArts.EAapp
```
Comment (Trenly) [inline manifests/e/ElectronicArts/EAapp/12.218.0.5475/ElectronicArts.EAapp.yaml:4]:
```suggestion
PackageIdentifier: ElectronicArts.EADesktop
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**